src/persist-txn/src/txns.rs

```diff
@@ -707,6 +722,23 @@ where
     }
 }
 
+#[cfg(test)]
+impl<K, V, T, D, O, C> TxnsHandle<K, V, T, D, O, C>
```

Comment:
super nit: can we put this next to the definition of `TestTxn` (inside the big `#[cfg(test)]mod tests` block) instead?

I have an irrational dislike for `#[cfg(test)]` gated `use` imports at the top of the file :)

Reply:
Done